server_apps/data_transfer/ResourceCenters/pullFromEZRC.pl

```diff
@@ -102,7 +102,7 @@ ($$)
     my $resourceCenter = $_[1];
 
     if  ($resourceCenter eq "EZRC"){
-	if (system("/local/bin/wget http://www.ezrc.kit.edu/downloads/$filename")) {
+	if (system("/local/bin/wget https://www.ezrc.kit.edu/downloads/$filename --no-check-certificate")) {
```

Comment:
Is the "--no-check-certificate" meant to be a permanent change, or do we want some type of reminder to go back and change it at some point?

Reply:
Good question. I don't know. We may ask Ryan M. how to handle this in the longer run.